frontend/src/layouts/SqlEditorLayout.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div class="relative h-screen overflow-hidden flex flex-col">
+    <template v-if="isDemo">
+      <BannerDemo />
+    </template>
+    <template v-else-if="isReadonly">
+      <div
+        class="h-12 w-full text-lg font-medium bg-yellow-500 text-white flex justify-center items-center"
+      >
+        The server is in read-only mode. You can Execute SQL queries us the
```

Comment:
I mean even server is NOT in readonly mode, SQL editor can only execute SELECT.  So the phase here is a bit misleading since user think they can execute update/delete/insert in non-readonly mode.